op-service/txmgr/txmgr.go

```diff
@@ -20,9 +20,14 @@ import (
 	"github.com/ethereum-optimism/optimism/op-service/txmgr/metrics"
 )
 
-// Geth defaults the priceBump to 10
-// Set it to 15% to be more aggressive about including transactions
-const priceBump int64 = 15
+const (
+	// Geth defaults the priceBump to 10
+	// Set it to 15% to be more aggressive about including transactions
+	priceBump int64 = 15
```

Comment:
We should consider reducing this to 10 (the min required by geth) with this change. I don't see a good reason to go above the minimum since if it's still too low we'll end up retrying again anyway with another bump.